MapleServer2/Types/Player.cs

```diff
@@ -32,12 +35,9 @@ public class Player
         public int JobId => JobGroupId * 10 + (Awakened ? 1 : 0);
 
         // Mutable Values
+        public Exp Exp { get; private set; }
```

Comment:
Is this used?

Reply:
Yes. `pWriter.WriteLong(player.Exp.PrestigeExp); // PrestigeExp` in PrestigePacket

Reply:
And other places.